### PR TITLE
Create error samples

### PR DESCRIPTION
**Pull request by KenzaYahia**
# Documentation

This PR adds samples for different types of errors.
List of added errors : 

- `ACCOUNT_TYPE_NOT_SUPPORTED`
- `CURRENCY_NOT_SUPPORTED`
- `NO_CHECKING_ACCOUNT`
- `NOT_ENOUGH_TRANSACTIONS`
- `OWNERSHIP_NOT_SUPPORTED`
- `USAGE_NOT_SUPPORTED`

### CODE REVIEW

File: samples/fr/README.md [hunk 1]
```diff
@@ -182,3 +182,33 @@
 - Has light overdraft at end of month
 - Has standard expenses: power, telecom, transport, multimedia, gym
 - Has savings
+
+## Account type not supported
+
```
Comment (g-ongenae) [inline samples/fr/README.md:187]: For coherency with the rest of the README, adding the language and table would be better:

```suggestion
**Language**: French 🇫🇷

| Format                       | Link                                                                                                                                           |
| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
| Algoan                       | [🔗](https://raw.githubusercontent.com/algoan/fake-open-banking-data/main/samples/fr/<name>.json)                                      |
| Budget Insight               | [🔗](https://raw.githubusercontent.com/algoan/fake-open-banking-data/main/raw-data/budget_insight_v2_0/fr/<name>.json)                 |
| Linxo Connect Direct Account | [🔗](https://raw.githubusercontent.com/algoan/fake-open-banking-data/main/raw-data/linxo_connect_direct_account_api_v3/fr/<name>.json) |
```

**Review by g-ongenae — COMMENTED** [review]
Except for my comment on the README, LGTM.

File: samples/fr/account_type_not_supported.json [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+          "dates": {
+            "debitedAt": "2024-04-07T12:00:00.000Z"
+          },
+          "description": "PRLV SEPA CREDIPAR Motif : 101M3323222 32 VOTRE IDEN",
+          "amount": -399.89
+        }
+      ],
+      "balance": 2200,
+      "balanceDate": "2023-08-31T12:00:00.000Z",
+      "iban": "FR195226475633NGNDOGOL7KZ15",
```
Comment (amarlankri) [inline samples/fr/account_type_not_supported.json:144]: You use the same IBAN for ownership not supported. Could you use a different one please?

File: samples/fr/account_type_not_supported.json [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+          "dates": {
+            "debitedAt": "2024-04-07T12:00:00.000Z"
+          },
+          "description": "PRLV SEPA CREDIPAR Motif : 101M3323222 32 VOTRE IDEN",
+          "amount": 399.89
+        }
+      ],
+      "balance": 1457.16,
+      "balanceDate": "2023-08-31T12:00:00.000Z",
+      "iban": "FR1248145844888DDFQXKV7BY13",
```
Comment (amarlankri) [inline samples/fr/account_type_not_supported.json:48]: This one is used 5 times 🙈

File: samples/fr/README.md (hunk 1 above)
Comment (KenzaYahia) [reply samples/fr/README.md:187]: So far i only added the algoan format, should i add BI and Linxo as well ? @g-ongenae

Comment (g-ongenae) [reply samples/fr/README.md:187]: They will be created automatically by the script: https://github.com/algoan/fake-open-banking-data/blob/main/scripts/converter.ts

**Review by g-ongenae — APPROVED** [review]

**Review by amarlankri — APPROVED** [review]
Thanks @KenzaYahia!

**Pull request merged**